mllib/src/main/scala/org/apache/spark/mllib/fpm/FPGrowth.scala

```diff
@@ -62,13 +62,14 @@ class FPGrowthModel[Item: ClassTag](val freqItemsets: RDD[FreqItemset[Item]]) ex
 @Experimental
 class FPGrowth private (
     private var minSupport: Double,
-    private var numPartitions: Int) extends Logging with Serializable {
+    private var numPartitions: Int,
+    private var mineSequences: Boolean) extends Logging with Serializable {
```

Comment:
`itemsets` and `sequences` are correct. The issue is that we have `freqItemsets` in the model, no matter whether it stores unordered sets or sequences. This may be confusing to users. I recommend renaming this variable to something like `ordered` or `preservesOrdering`. cc @jkbradley